Cinder_Compatible_API_Design.md

```diff
@@ -0,0 +1,159 @@
+# Cinder-compatible api design
+
+Status: Pending
+
+Version: Alpha
+
+Author(s): PengYi
+
+## Summary
+
+Cinder-compatible api receives cinder api request and convert it to hotpot api. 
+
+## Motivation
+
+To eliminate the differences between cinder api and hotpot api, an adapter(cinder-compatible api) should receive cinder api request and convert it to hotpot api. 
+
+### Goals
+
+* The adapter(cinder-compatible api) shouldn’t have an impact on the current openstack or hotpot environment. It also provides rest api to counterfeit the cinder api.
+
+### Non-Goals
+
+The proposal only satisfies the data model in the current OpenSDS. 
+
+## Proposal
+The working steps of Cinder-compatible api are as follows:
+1.Users use a python-cinderclient to issue an api request.
+2.Cinder-compatible api receive the request, parse and convert the request to hotpot. 
+3.Hotpot receive the request and performs related operations and returns the result to the Cinder-compatible api.
+3.Cinder-compatible api receive the result, parse and convert the result to python-cinderclient. 
```

Comment:
s/3/4

The numbering does not look right in the printed doc.  Please fix them.

https://github.com/PengYiOfOpenSDS/design-specs/blob/5d5aa6246427b22a33d582295ed248ebc667dc47/Cinder_Compatible_API_Design.md